src/deployment/deployment.ts

```diff
@@ -153,7 +153,8 @@ export class Deployment {
     // TODO: allocation is not used in this api?
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     const allocation = await this.modules.payment.createAllocation({
-      budget: 1,
+      budget: 1, // TODO: Shouldn't this be generated by the deployment internally when the demand is known?
+      expirationSec: 24 * 60 * 60,
```

Comment:
not a fan of hard-coding this, can't we do `demand.market.rentHours * 60 * 60`?

Reply:
It actually could be entirely dropped (this line), as the allocation is not used in that file at any case.